content/hacking-atom/sections/hacking-on-atom-core.md

```diff
@@ -83,33 +83,32 @@ In order to build Atom from source, you need to have a number of other requireme
 ##### Requirements
 
 * macOS 10.9 or later
-* Node.js 6.x or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
-* npm 3.10.x or later (run `npm install -g npm`)
-* Python v2.7.x
+* Node.js 10.12 or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
+* npm 6 or later (run `npm install -g npm`)
+* Python v2.7.x or 3.5+
 * Command Line Tools for [Xcode](https://developer.apple.com/xcode/downloads/) (run `xcode-select --install` to install)
 
 {{/mac}}
 
 {{#windows}}
 
-* Node.js 6.9.4 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
-* Python v2.7.x
+* Node.js 10.12 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
```

Comment:
We use Node 12 in our CI builds. Shouldn't that be the supported version?

Reply:
10 is still an officially supported Long Term Support version for now: https://nodejs.org/about/releases/

(Until the end of April in 2021)